### PR TITLE
add results

### PR DESCRIPTION
**Pull request by hbrkic**
Results from Hrvoje. Plan 1 and 3

### CODE REVIEW

**Comment by grzanka** [comment]
@hbrkic what are the units of scored quantities ? Can you add for a bookkeeping the input files as well ?

**Review by grzanka — APPROVED** [review]

File: data/mcnp/results_HB/plan_1_2/HB_Z_narrow_dose_p1.dat [hunk 1]
```diff
@@ -0,0 +1,206 @@
+-10.2 1.15E-02 0.00000229572
```
Comment (nbassler) [inline data/mcnp/results_HB/plan_1_2/HB_Z_narrow_dose_p1.dat:1]: too few digits maybe on the second column?

SH12A has:
-2.20063 0.0102269 5.4457e-06
-2.10188 0.0102531 5.26708e-06
-2.00313 0.0102735 5.43075e-06
-1.90438 0.0102924 5.58671e-06
-1.80563 0.0103071 5.75597e-06

**Review by nbassler — COMMENTED** [review]
"plan_1_1" is that "plan_1a"?

I would prefer to be super strict when naming the plans to avoid confusion

File: data/mcnp/results_HB/plan_1_2/HB_Z_narrow_dose_p1.dat (hunk 1 above)
Comment (nbassler) [reply data/mcnp/results_HB/plan_1_2/HB_Z_narrow_dose_p1.dat:1]: what i mean is if you look there is very little progression the first points, dose increases in discrete steps, and the error bar is way smaller than the number of digits on the result.

Comment (hbrkic) [reply data/mcnp/results_HB/plan_1_2/HB_Z_narrow_dose_p1.dat:1]: I see what is the issue, with too few digits. this is corrected now. 

Regarding the naming:
plan_1_1 - is a plan 1 simulated with the phase space file plan_1_1
....

**Review by nbassler — APPROVED** [review]

**Comment by nbassler** [comment]
remaining issues will be addressed in #79

**Pull request merged**